public/app/plugins/datasource/stackdriver/components/StackdriverPicker.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
This component could probably be reused be other datasources, e.g azure monitor. In that case we need to:  
* give it a more generic name
* move it to `app/core/components/Select/` 
* move the angular directive registration from the stackdriver query editor to `app/core/angular_wrappers`. 

Good or bad idea?

Reply:
renamed it to MetricSelect. also moved it to `app/core/components/Select/`. otherwise it cannot be used by other datasources right?

Reply:
hmm all other select components ends with Picker. DatasourcePicker, TeamPicker. Maybe it should be MetricPicker after all?